bcda/models/models.go

```diff
@@ -480,17 +480,9 @@ func (cclfBeneficiary *CCLFBeneficiary) GetBlueButtonID(bb client.APIClient) (bl
 	if cclfBeneficiary.BlueButtonID != "" {
 		return cclfBeneficiary.BlueButtonID, nil
 	}
-	var oldRecord CCLFBeneficiary
 	db := database.GetGORMDbConnection()
```

Comment:
Not sure if we need a db connection at all with the query being removed??

Reply:
Good eye!  Removed in latest commit.